### PR TITLE
[FSDE-17717] Html blueprint for Edit ribbon

### PR DESCRIPTION
**Pull request by jllr**
- [x] Added Save view modes
[ ] Base
[ ] Save
[ ] Edit
[ ] Edit w/ delete
- [x] Added tests for Save view modes
- [x] Fixed Toolbar base test sizing

Link to Jira card
https://ctl.atlassian.net/browse/FSDE-17717

Link to Adobe XD designs
https://xd.adobe.com/view/79ce720b-3477-429c-9fae-5836af3a7d3a-2c9d/screen/19d76972-c597-4a50-8a08-c2dde3d6ce12/specs/

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://49055c4265ff24d722724a3638d32b2223bda5d3-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://54428a3208f4ea83cdca9cfc7ffb13097a1a1116-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline has run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://bcbb53a4cb33a5f03ddd6acdb2451311c01354fb-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/chi/components/save-view/save-view.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      margin-left: 0.5rem;
+
+      .chi-save-view__label {
+        font-weight: 600;
+      }
+    }
+  }
+
+  &__save {
+    border: 0.0625rem dashed $save-view-save-border-color;
```
Comment (mattnickles) [inline src/chi/components/save-view/save-view.scss:39]: In addition to the color can we theme the border style?

File: src/chi/components/save-view/save-view.scss [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+      + .chi-button {
+        margin-left: 0.5rem;
+      }
+    }
+  }
+
+  &__ribbon {
+    align-items: center;
+    background-color: #0075c9;
```
Comment (mattnickles) [inline src/chi/components/save-view/save-view.scss:75]: We'll need to create a variable e.g. $save-view-ribbon-background-color and apply it both here and the arrow after border color value. $color-background-primary should work as the variable value for all themes except Brightspeed.

**Comment by SCMAUTO** [comment]
The CI pipeline has run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://78a77b01ad07a8b133d6e3327f0b8d4209c499be-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/5/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline has run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in http://b422fd01ef2ebdb096c1b7d537f291a685bd6c82-chi-pr-po-ctl.s3-website-us-east-1.amazonaws.com (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by janibolkvadze — APPROVED** [review]

File: src/chi/themes/brightspeed/_variables.scss [hunk 3]
```diff
@@ -1316,6 +1316,22 @@ $range-slider-disabled-opacity:             $opacity-60;
...
+$save-view-divider-height:                  1.25rem;
+$save-view-results-font-size:               $font-size-md;
+$save-view-label-font-size:                 $font-size-md;
+$save-view-label-font-weight:               $font-weight-semi-bold;
+$save-view-save-border-color:               $color-border-info;
+$save-view-save-border-style:               dashed;
+$save-view-ribbon-background-color:         $color-blue-70;
+$save-view-ribbon-label-color:              $color-text-white;
+$save-view-ribbon-label-font-style:         italic;
+$save-view-ribbon-arrow-border-color:       $color-blue-70;
```
Comment (mattnickles) [inline src/chi/themes/brightspeed/_variables.scss:1332]: We can't render the Lumen blue color in the Brightspeed theme. I would recommend using black.

```suggestion
$save-view-ribbon-background-color:         $color-black;
$save-view-ribbon-label-color:              $color-text-white;
$save-view-ribbon-label-font-style:         italic;
$save-view-ribbon-arrow-border-color:       $color-black;
```

**Review by mattnickles — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline has run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in http://228745c8323b52575dfa900094366f16c8bb1e83-chi-pr-po-ctl.s3-website-us-east-1.amazonaws.com (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline has run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in http://8e6f66540878acd97499a0a9962e9c6f0152934f-chi-pr-po-ctl.s3-website-us-east-1.amazonaws.com (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by mattnickles — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/9/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline has run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-388/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in http://e8f3304dc7d3eac555fe862e02ae37f8082301de-chi-pr-po-ctl.s3-website-us-east-1.amazonaws.com (external) and in https://nginx-pr-388-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Pull request merged**